src/coreclr/jit/block.h

```diff
@@ -2497,7 +2501,7 @@ class AllSuccessorEnumerator
 
 public:
     // Constructs an enumerator of all `block`'s successors.
-    AllSuccessorEnumerator(Compiler* comp, BasicBlock* block, const bool useProfile = false);
+    AllSuccessorEnumerator(Compiler* const comp, BasicBlock* const block);
```

Comment:
Nit: marking parameters as const in declarations is not very beneficial (it is a property of the definition, not of the declaration)

Reply:
We have a number of places where we have these `const`. IMO we should enable this rule of clang-tidy: https://clang.llvm.org/extra/clang-tidy/checks/readability/avoid-const-params-in-decls.html

Reply:
> We have a number of places where we have these const. IMO we should enable this rule of clang-tidy:

That seems reasonable. I can open a PR for that.